app/controllers/TelephoneNumberController.scala

```diff
@@ -67,11 +70,26 @@ class TelephoneNumberController @Inject()(
       Ok(view(preparedForm, mode, index, trusteeName))
   }
 
+  private def getMessagePrefix(index: Int, request: DataRequest[AnyContent]) = {
+    val isLead = request.userAnswers.get(IsThisLeadTrusteePage(index)).get
```

Comment:
@jassalrichy You're doing  a `.get` here on the Option without first checking it is defined.
Can we add a `requiredAnwer` to the actions and redirect if the answer does not exist?

Reply:
requiredAnswer check added